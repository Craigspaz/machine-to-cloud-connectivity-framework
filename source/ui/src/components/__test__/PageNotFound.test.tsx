// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

import { render } from '@testing-library/react';
import PageNotFound from '../PageNotFound';

test('renders the PageNotFound component', async () => {
  const pageNotFound = render(<PageNotFound />);
  expect(pageNotFound.container).toMatchSnapshot();
});
