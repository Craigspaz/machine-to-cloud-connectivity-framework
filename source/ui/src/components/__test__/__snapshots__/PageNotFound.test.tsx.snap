// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the PageNotFound component 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="row"
    >
      <div
        class="col"
      >
        <div
          class="jumbotron"
        >
          <h3>
            page.not.found
            : 
            <code>
              /
            </code>
          </h3>
        </div>
      </div>
    </div>
  </div>
</div>
`;
